.circleci/config.yml

```diff
@@ -1,37 +1,63 @@
 version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.4.6
 jobs:
-  test:
+  testold:
     parameters:
         docker_image:
           type: string
-          default: circleci/node:current-browsers
+          default: cimg/node:current-browsers
     docker:
       - image: << parameters.docker_image >>
     steps:
       - checkout
       - run:
           name: Setup
+          command: |
+            rm -rf node_modules package-lock.json
+            npm install
+      - run:
+          name: Running all unit tests
           command: |
             node -v
             npm -v
+            CHROME_BIN=/usr/bin/google-chrome npm run test
+  test:
+    parameters:
+        docker_image:
+          type: string
+          default: cimg/node:current-browsers
+    docker:
+      - image: << parameters.docker_image >>
+    steps:
+      - checkout
+      - browser-tools/install-browser-tools
+      - run:
+          name: Setup
+          command: |
             rm -rf node_modules package-lock.json
             npm install
       - run:
           name: Running all unit tests
           command: |
-            npm run test
+            node -v
+            npm -v
+            CHROME_BIN=/usr/bin/google-chrome npm run test
 
 workflows:
   version: 2
   test-all-node-versions:
     jobs:
-      - test:
-          docker_image: circleci/node:8-browsers
       - test:
           docker_image: circleci/node:10-browsers
-      - test:
+      - testold:
```

Comment:
why testing node v12 is `testold`?

Reply:
v12 does not have any ESM module support